internal/v1/genny/resource/templates/use_model/actions/resource-name.go.tmpl

```diff
@@ -132,9 +132,9 @@ func (v {{.opts.Name.Resource}}Resource) Create(c buffalo.Context) error {
 
       return c.Render(http.StatusUnprocessableEntity, r.HTML("/{{.opts.Name.File.Pluralize}}/new.plush.html"))
     }).Wants("json", func (c buffalo.Context) error {
-      return c.Error(http.StatusUnprocessableEntity, r.JSON({{.opts.Model.VarCaseSingle}}))
+      return c.Render(http.StatusUnprocessableEntity, r.JSON({{.opts.Model.VarCaseSingle}}))
     }).Wants("xml", func (c buffalo.Context) error {
-      return c.Error(http.StatusUnprocessableEntity, r.XML({{.opts.Model.VarCaseSingle}}))
+      return c.Render(http.StatusUnprocessableEntity, r.XML({{.opts.Model.VarCaseSingle}}))
```

Comment:
Shouldn't these be rendering the errors, not the widget?

Reply:
```
return c.Render(http.StatusUnprocessableEntity, r.JSON(verrs))
```

Reply:
My only idea there was to provide a response in the same language of the request that's being done to the endpoint. IDK if by setting the error Buffalo would automatically handle JSON/XML to render the error.

Reply:
that was correct with `Auto`, and true of `HTML`. while you're returning an error status code, you're never doing anything with the validation errors. the user just sees their submitted data back again.

Reply:
Oh! I just saw that comment with the example. I will take that path:

return c.Render(http.StatusUnprocessableEntity, r.JSON(verrs))